src/csharp/Microsoft.Spark/Sql/Types/Timestamp.cs

```diff
@@ -94,7 +94,7 @@ public Timestamp(DateTime dateTime)
         /// <summary>
         /// Readable string representation for this type.
         /// </summary>
-        public override string ToString() => _dateTime.ToString("yyyy-MM-dd HH:mm:ss.ffffff");
+        public override string ToString() => _dateTime.ToString("yyyy-MM-dd HH:mm:ss.ffffffZ");
```

Comment:
I assume this is a bug ?  Converting to string and casting back to Timestamp in Spark caused the time to shift 8 hours.
@elvaliuliuliu @imback82

Reply:
Adding a separate PR instead to address this.  https://github.com/dotnet/spark/pull/871

Reply:
thanks. let's discuss this in #871 and remove from this PR.